packages/did-core-test-server/suites/implementations/dereferencer-cheqd.json

```diff
@@ -0,0 +1,161 @@
+          "error": "invalidDidUrl",
+          "retrieved": "2022-05-11T14:54:20Z",
+          "did": {
+            "didString": "did_cheqd_mainnet_zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY",
+            "methodSpecificId": "did_cheqd_mainnet_zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY",
+            "method": "cheqd"
+          }
+        },
+        "contentStream": "",
+        "contentMetadata": {
+        }
+      }
+    },
+    {
+      "function": "dereference",
+      "input": {
+        "didUrl": "did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWku",
+        "dereferenceOptions": {
+          "accept": "application/did+ld+json"
+        }
+      },
+      "output": {
+        "dereferencingMetadata": {
+          "contentType": "application/did+ld+json",
+          "error": "notFound",
+          "retrieved": "2022-05-11T14:31:02Z",
+          "did": {
+            "didString": "did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWku",
+            "methodSpecificId": "zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWku",
+            "method": "cheqd"
+          }
+      
+        },
+        "contentStream": "",
+        "contentMetadata": {
+        }
+      }
+    },
+    {
+      "function": "dereference",
+      "input": {
+        "didUrl": "did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY",
+        "dereferenceOptions": {
+          "accept": "application/did+ld+json"
+        }
+      },
+      "output": {
+        "dereferencingMetadata": {
+          "contentType": "application/did+ld+json",
+          "retrieved": "2022-05-11T14:45:33Z",
+          "did": {
+            "didString": "did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY",
+            "methodSpecificId": "zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY",
+            "method": "cheqd"
+          }
+        },
+        "contentStream": "{\"@context\": \"https://www.w3.org/ns/did/v1\",\"id\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY\",\"authentication\":[\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#key1\"],\"service\":[{\"id\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#website\",\"serviceEndpoint\":\"https://www.cheqd.io\",\"type\":\"LinkedDomains\"},{\"id\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#non-fungible-image\",\"serviceEndpoint\":\"https://gateway.ipfs.io/ipfs/bafybeihetj2ng3d74k7t754atv2s5dk76pcqtvxls6dntef3xa6rax25xe\",\"type\":\"LinkedDomains\"},{\"id\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#twitter\",\"serviceEndpoint\":\"https://twitter.com/cheqd_io\",\"type\":\"LinkedDomains\"},{\"id\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#linkedin\",\"serviceEndpoint\":\"https://www.linkedin.com/company/cheqd-identity/\",\"type\":\"LinkedDomains\"}],\"verificationMethod\":[{\"controller\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY\",\"id\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#key1\",\"publicKeyMultibase\":\"zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkYsWCo7fztHtepn\",\"type\":\"Ed25519VerificationKey2020\"}]}",
```

Comment:
pretty sure `LinkedDomains` and `Ed25519VerificationKey2020` are not in `https://www.w3.org/ns/did/v1`

Reply:
Within https://www.w3.org/ns/did/v1 it explains that the [service](https://www.w3.org/TR/did-core/#dfn-service) type and its associated properties SHOULD be registered in the DID Specification Registries [[DID-SPEC-REGISTRIES](https://www.w3.org/TR/did-core/#bib-did-spec-registries)]

We perhaps should make explicit reference to the DID Spec Registries in the context, where LinkedDomains is one of only two Service Types currently registered: https://www.w3.org/TR/did-spec-registries/#linkeddomains